UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/appsettings.json

```diff
@@ -34,6 +34,7 @@
   },
   "SalesCatalogue": {
     "BaseUrl": "",
+    "PublicBaseUrl": "",
```

Comment:
I think `PublicBaseUrl` should have been added to the `FileShareService` section, not the `SalesCatalogue` section